dashboard/src/app/index.module.ts

```diff
@@ -210,7 +212,7 @@ angular.element(document).ready(() => {
   }).catch((error: string) => {
     console.error(`Can't GET "/api". ${error ? 'Error: ' : ''}`, error);
     if (!hasCertificateError) {
-      error = `${error}<div class="error-body">Click <a href="/">here</a> to reload page.</div>`
+      error = `${error}<div>Click <a href="/">here</a> to reload page.</div>`
```

Comment:
Che logo seems to be missing. See 
![image](https://user-images.githubusercontent.com/52013984/74865677-82192900-531f-11ea-99fa-ab5d950cc0a6.png)
from https://github.com/eclipse/che/issues/16065#issuecomment-588386383

Reply:
done